### ISSUE TITLE
Have band details in database

### ISSUE THREAD

**Issue by danieljames-dj** [thewca/worldcubeassociation.org#8159]
Currently, we have the fields like per_competitor_dues in each band and PERCENT_REGISTRATION_FEE_USED_FOR_DUE_AMOUNT hard-coded in https://github.com/thewca/worldcubeassociation.org/blob/master/WcaOnRails/app/models/country_band.rb.

It will be good if we have a table named something like `band_details` and have the above two band details there.

Note: PERCENT_REGISTRATION_FEE_USED_FOR_DUE_AMOUNT is currently 0.15 for all bands, but it can be different for different bands in future.

In the above mentioned file, if we have the values fetched from the table, it will be really easy for both WST & WFC for any future changes.

**Comment by EdHollingdale** [comment]
Can we update prioritisation for this one? I think there are two sub tasks:
1. make it so that PERCENT_REGISTRATION_FEE_USED_FOR_DUE_AMOUNT is variable by band. As of 1 Jan 2024 with the updated policy this amount is variable (0% for Band 0, 5% for Bands 1 and 2, and 15% for Bands 3-5). 
2. make it so that per_competitor_dues and PERCENT_REGISTRATION_FEE_USED_FOR_DUE_AMOUNT are in the database rather than being hardcoded (lower priority, since we don't currently have plans to change them in the next two years at least)

Note, I have done a minor change to the Dues calculation disclaimer to warn that the calculation is incorrect currently (#8704) which will have to be reversed once 1. is complete (since that will fix the bug).

**Comment by danieljames-dj** [comment]
I've fixed 1 here: https://github.com/thewca/worldcubeassociation.org/pull/8705. This will also predict dues with correct band in our WFC export, so that we won't have a warning there.

For 2, I would suggest having a table `band_details` with following columns:
- band_number
- per_competitor_dues
- competition_percent_dues
- start_date
- end_date

This way, we can store the historical change in dues values as well. @gregorbg could you please approve if the above columns will be fine for long term fix?

Since this is a low-priority change, I would suggest some volunteers to take it up once Gregor approves the above model.
